docs/topics/forms/formsets.txt

```diff
@@ -274,6 +274,8 @@ is where you define your own validation that works at the formset level::
     ...             return
     ...         titles = []
     ...         for form in self.forms:
+    ...             if self.can_delete and self._should_delete_form(form):
+    ...                 continue
     ...             title = form.cleaned_data['title']
```

Comment:
I think the second change from ticket description is still valid, i.e. `form.cleaned_data` may not contain `'title'` key:
```suggestion
     ...             title = form.cleaned_data.get('title')
```

Reply:
Absolutely, I overlooked that change. Fixed.